spec/mailers/map_mailer_spec.rb

```diff
@@ -1,10 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe MapMailer, type: :mailer do
-  let(:map) { create(:map) }
-  let(:inviter) { create(:user) }
-  let(:invitee) { create(:user) }
+  describe 'access_request_email' do
+    let(:request) { create(:access_request) }
+    let(:map) { create(:map) }
+    let(:mail) { described_class.access_request_email(request, map) }
+
+    it { expect(mail.from).to eq ['team@metamaps.cc'] }
+    it { expect(mail.to).to eq [map.user.email] }
+    it { expect(mail.subject).to match map.name }
+    it { expect(mail.body.encoded).to match map.name }
+    it { expect(mail.body.encoded).to match request.user.name }
+    it { expect(mail.body.encoded).to match 'Approve Request' }
```

Comment:
It's in there somewhere! I'm not sure why. I'd have to go back to the templates. But they do pass right now